### ISSUE TITLE
MP4 previews generated but not shown

### ISSUE THREAD

**Issue by labuzer** [nextcloud/previewgenerator#335]
Hello

I've recently moved to a new server and thus have reinstalled Nextcloud 25. I have setup my server identical to before, pretty much copying the settings. My previous server generates movie previews with no issue, and so does my new setup. However, although they get generated they are not displayed within Nextcloud. I am unsure why this is, I am running a fresh install with PHP 8.1 and the newest Nextcloud version. I am not seeing any errors anywhere that could indicate what is wrong.

Anyone has any idea what could be the cause of this?

**Comment by st3iny** [comment]
Do you have ffmpeg installed?

**Comment by labuzer** [comment]
> Do you have ffmpeg installed?

Yes I do. I don't think the previews would be generated without it being installed.

**Comment by Guttalax** [comment]
Same problem for me, config ok, ffmpeg and imagemagik, no errors trown when generating previews but still can't see them.
Additional info:
Previews work only in trash folder.

**Comment by st3iny** [comment]
That is very interesting. Thanks for following up.

I'm going to try to reproduce it locally with ffmpeg.

**Comment by joshtrichards** [comment]
Is this like #355, where they're generated but not displayed other than in the trash bin? Can those of your reporting this confirm if you see the same as what's reported in that issue?
